packages/Testing/PHPUnit/AbstractRectorTestCase.php

```diff
@@ -125,6 +125,11 @@ protected function doTestFile(string $fixtureFilePath): void
         // write temp file
         FileSystem::write($inputFilePath, $inputFileContents);
 
+        if (! is_file($inputFilePath)) {
+            // give enough time to write process
+            sleep(3);
+        }
```

Comment:
on file not exists, it probably file write and read is overlapped, this is to ensure give enough time to write process.